core/src/main/java/com/oath/oak/OakWBuffer.java

```diff
@@ -22,14 +22,6 @@
      */
     int capacity() throws NullPointerException;
 
-    /**
-     * Going to be deprecated!!!!!!!!!
-     *
-     * @return the actual ByteBuffer
-     */
-    @Deprecated
-    ByteBuffer getByteBuffer();
```

Comment:
IT WONT COMPILE WITH DRUID!!!! PUT IT BACK! We should first resolve how to work without in in Druid....

Reply:
I put it back